_posts/0000-01-12-ukbill.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Well I'll be buggered!
```

Comment:
### **@ukbill, This is clearly your best work!  Great job.  Go Celebrate!**

Reply:
Thank you for your kind words